### DISCUSSION TITLE
Why does the pass element in the Topological Consistency Descriptive Results always have to be false?

### DISCUSSION THREAD

**Discussion by PB-GNM** [INSPIRE-MIF/helpdesk-validator#792] [Q&A] [answered]
Dear helpdesk of the INSPIRE validator,

I have the following question:
Why does the pass element in the Topological Consistency Descriptive Results always have to be false?

This is part of test md isdss 2.8 .
In https://github.com/inspire-eu-validation/metadata/blob/2.0/isdss/topological-consistency-descriptive-results.md it is stated that the pass element should always be false.

Best regards,
Pieter Bresters,
Geonovum,
Netherlands

**Comment by fabiovinci** [comment, marked as answer]
Dear @PB-GNM,

since the related metadata element is conditional, i.e. "_it is **mandatory only if** the data set includes types from the Generic Network Model and **does not assure centreline topology** (connectivity of centrelines) for the network_", 

![image](https://user-images.githubusercontent.com/22212406/174591154-3b099db4-fa93-434d-8711-57e059b8602a.png)

if it is present, it means that "_the data set does **not assure the centreline topology** for the network_", for this reason, the `gmd:pass` element is `false` by default. 

![image](https://user-images.githubusercontent.com/22212406/174590988-958d217e-ccc7-4569-9dd3-88fc217525e8.png)

I hope that this helps to clarify.

**Comment by PB-GNM** [comment]
Dear Fabio,

Thank you for the quick answer!
To check, if I understand it correctly:
Does this mean we should not use the DQ_TopologicalConsistency element if centrelines topology is assured for the network?

Best regards,
Pieter Bresters,
Geonovum,
Netherlands

**Comment by fabiovinci** [comment]
Dear @PB-GNM,

> Does this mean we should not use the DQ_TopologicalConsistency element if centrelines topology is assured for the network?

Yes, it is correct.

**Comment by PB-GNM** [comment]
Thank you for the clear answer.
